src/codegen_type.cpp

```
@@ -21,6 +21,7 @@ llvm::Type *CodeGen::resolveType(const std::string &typeName) {
     if (typeName == "str")   return ptrTy_;
     if (typeName == "Error") return errorTy_;
     if (typeName == "any")   return anyTy_;
+    if (typeName == "Type")  return typeTy_;
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)